### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RhehanMuzaky**
add new text in line 3 & 4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # learn-collaboration
-I want a learn squash & merge.  
-So i create some commit in this file.  
-This is very interesting.  
+Learn Github at Dicoding is really fun!.
```
Comment (rhehan7) [inline README.md:2]: you haven't added <br> / 2 spaces  to create a new line in this row, nor in the following rows

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+Documenting with Github becomes easier.
```
Comment (rhehan7) [inline README.md:3]: new line?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
+Code management made easier because of Github.
```
Comment (rhehan7) [inline README.md:4]: new line?

**Review by rhehan7 — CHANGES_REQUESTED** [review]
need to adjust the changes you have made

**Comment by RhehanMuzaky** [comment]
I just finished the repair sir.

File: README.md
Comment (rhehan7) [inline README.md:1]: Changes accepted, thankyou rey.

**Review by rhehan7 — APPROVED** [review]
Changes accepted

**Comment by rhehan7** [comment]
Thanks for contributing to this repository @RhehanMuzaky

**Pull request merged**